apps/users/templates/users/t-shirt.html

```diff
@@ -0,0 +1,137 @@
+{% extends "impala/base_shared.html" %}
+{% from 'includes/forms.html' import pretty_field, required %}
+
+{% block title %}{{ page_title(_('T-shirt Order')) }}{% endblock %}
+
+{% block bodyclass %}t-shirt-form{% endblock %}
+
+{% block content %}
+<h1>{{ _('Special Edition Add-on Creator T-shirt') }}</h1>
```

Comment:
Well, I copied the initial version of this from another `users` template. None of the content blocks are indented from their template tags here, so I figured I should stick with the same style.